Commands/notify.py

```diff
@@ -142,7 +143,7 @@ async def wait_for_added_reactions(self, ctx: commands.Context, msg_id: int, gui
                 reaction, user = await ctx.bot.wait_for(
                     "reaction_add",
                     check=lambda emoji, author: emoji.message.id == msg_id and not author.bot,
-                    timeout=30.0,
+                    timeout=timeout,
```

Comment:
timeout only gets defined later in this file and not really as a constant, should we not put this in a config or a constant somewhere? If this function gets called anywhere else it would have an issue with the non-existing timeout.

Reply:
Bob, read the code right, it's right there in the arguments of the function you idiot.